feat(changelog): environmental-footprint-added-discover-the-environmenta 2026-04-29 (#6472)
scaleway/docs-content: changelog/april2026/2026-04-29-environmental-footprint-added-discover-the-environmenta.mdx

---

> title: Discover the environmental impact of your Managed Database now!

Reviewer: ```suggestion
title: Discover the environmental impact of your Managed Database
```

status: added
date: 2026-04-29
category: billing
product: environmental-footprint
---

> Our Environmental Footprint Calculator now supports Managed Databse (PostgreSQL & MySQL, Redis and MongoDB) !

Reviewer: ```suggestion
Our Environmental Footprint Calculator now supports Managed Databases (PostgreSQL & MySQL, Redis, and MongoDB)
```

Designed to go beyond simple accounting, it provides real usage-based impact metrics, covering both CO₂ emissions and water consumption.

> This multi-criteria approach helps you understand, compare and reduce the footprint of your cloud infrastructure. [Learn more](https://www.scaleway.com/en/docs/environmental-footprint/additional-content/managed-databases-env-footprint/)

Reviewer: ```suggestion
This multi-criteria approach helps you understand, compare, and reduce the footprint of your cloud infrastructure. Learn more in our document about [Managed Database footprint calculation](/environmental-footprint/additional-content/managed-databases-env-footprint/).
```
